WIP - MTA-4637: Pattern matching docs need urgent improvements (#92)
migtools/mta-documentation: docs/topics/yaml-dotnet-provider.adoc

// Module included in the following assemblies:
//
// * docs/rules-development-guide/master.adoc

:_mod-docs-content-type: REFERENCE
[id="yaml-dotnet-provider_{context}"]
= Dotnet provider

The `dotnet` provider is an external provider used to analyze .NET and C# source code. Currently, the provider supports the `referenced` capability.

.`referenced`

> The `referenced` capability enables the provider to find references in the source code.

Reviewer: ```suggestion
By using the `referenced` capability, the provider finds references in the source code.
```
